src/styles/common/_tooltip.scss

```diff
@@ -22,3 +22,11 @@ $tooltip-width: 1;
     border-radius: 2px;
 }
 
+.select-disable {
```

Comment:
Maybe we could place this in britecharts/src/styles/base/_typography.scss as it will probably be used in other charts?

Reply:
Yes! I also added it to the big Tooltip (not just mini-tooltip) - see updated commit. Will add it to Typography.